samples/apps/copilot-chat-app/webapp/src/components/chat/ChatWindow.tsx

```diff
@@ -87,7 +87,7 @@ export const ChatWindow: React.FC = () => {
     const dispatch = useAppDispatch();
     const { conversations, selectedId } = useAppSelector((state: RootState) => state.conversations);
     const chatName = conversations[selectedId].title;
-    const [title, setTitle] = useState<string | undefined>(selectedId ?? undefined);
+    const [title = '', setTitle] = useState<string | undefined>(selectedId ?? undefined);
```

Comment:
removed the `?? undefined` since it was unneeded. with that said, `?? ''` would not produce a default value of `''` since `selectedId` is always truthy.